src/spec/payid-easy-checkout.md

```diff
@@ -0,0 +1,330 @@
+---
+coding: utf-8
+
+title: Draft 1: PayID Easy Checkout Protocol
+docname: payid-easy-checkout-protocol
+category: std
+
+pi: [toc, sortrefs, symrefs, comments]
+smart_quotes: off
+
+area: security
+author:
+  -
+    ins: N. Kramer
+    name: Noah Kramer
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: nkramer@ripple.com
+    uri: https://www.ripple.com
+  -
+    ins: I. Simpson
+    name: Ian Simpson
+    org: Ripple
+    street: 315 Montgomery Street
+    city: San Francisco
+    region: CA
+    code: 94104
+    country: US
+    phone: -----------------
+    email: isimpson@ripple.com
+    uri: https://www.ripple.com
+
+normative:
+    RFC2119:
+    RFC2818:
+    RFC8446:
+    RFC7231:
+    RFC7413:
+    RFC6265:
+    PAYID-URI:
+      title: "The 'payid' URI Scheme"
+      target: https://tbd.example.com/
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+    PAYID-DISCOVERY:
+      title: "PayID Discovery"
+      author:
+        ins: D. Fuelling
+        fullname: David Fuelling
+
+informative:
+    RFC5988:
+
+--- note_Feedback
+
+This specification is a draft proposal, and is part of the [PayID Protocol](https://payid.org/) initiative. Feedback related to this document should be sent in the form of a Github issue at: https://github.com/payid-org/rfcs/issues.
+ 
+--- abstract
+This specification defines the PayID Easy Checkout Protocol, which can be used to allow two parties to transact money in a formalized way for some means of doing business with one another.
+
```

Comment:
Thinking about this abstract -- I wonder if a better abstract would be something that says:

"This spec allows a payment recipient, such as a merchant or a non-profit, to automatically navigate a user to the user's wallet using only a PayID"

The current abstraction is somewhat vague on that point, and seems to imply that this RFC is helping to formalize a payment, but really it's just formalizing "get a user to his wallet, and also send along some payment details".